### PR TITLE
Model/AdaBoostStump

### PR DESCRIPTION
**Pull request by yvokeller**

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: AdaBoostStumpClassifier, 2023-05-19 16-38-09

## Hyperparameters

| Path        | Param                         | main        | workspace   |
|-------------|-------------------------------|-------------|-------------|
| params.yaml | train_test_split_ratio02.type | measurement | segment     |

## Metrics

| Path                 | Metric          | main    | workspace   | Change   |
|----------------------|-----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc  | 0.70765 | 0.5705      | -0.13715 |
| dvclive/metrics.json | test.epoch.f1   | 0.66619 | 0.5052      | -0.16099 |
| dvclive/metrics.json | train.epoch.acc | 0.72471 | 0.57708     | -0.14764 |
| dvclive/metrics.json | train.epoch.f1  | 0.68623 | 0.51858     | -0.16765 |

## Plots

### Confusion Matrix

Confusion Matrix

**Comment by gabrieltorresgamez** [comment]
what we doing with all of this

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-06-02 08-10-39

## Hyperparameters

| Path   | Param   | main   | workspace   |
|--------|---------|--------|-------------|

## Metrics

| Path   | Metric   | main   | workspace   | Change   |
|--------|----------|--------|-------------|----------|

## Plots

### Confusion Matrix

Confusion Matrix

**Comment by gabrieltorresgamez** [comment]
ok, now if it's merged it retains the code but uses the histgradientboostingclassifier in main, since it performs better

**Review by gabrieltorresgamez — APPROVED** [review]
